TelecomColourCode_Main.h

```diff
@@ -0,0 +1,56 @@
+/* ******************************************************************************
+ ***     \file        TelecomColourCode_Main.h
+ ***     \ingroup     Session1-TCQ
+ ***     \author      Rohith K N
+ ***
+ ***     \brief       Header file for the module
+ ***
+ ***     \par         File_description
+ ***                  This file have the declarations of the variables and methods of the module
+ ***
+ *****************************************************************************/
+#ifndef TELECOM_COLOR_CODE_MAIN_H_
+#define TELECOM_COLOR_CODE_MAIN_H_
+/*=============================================================================
+ =======                            INCLUDES                             =======
+ ============================================================================*/
+ /*------ standard includes -------*/
+#include <stdio.h>
+
+#define MAX_COLORPAIR 				25		/*maximum number of colour pairs possible*/
+
```

Comment:
`MAX_COLORPAIR` is defined as 25. Need to guess that it's the product of number of major and minor colors. Any way to clarify?